test_pytest_html.py

```diff
@@ -44,7 +44,17 @@ def assert_results_by_outcome(html, test_outcome, test_outcome_number,
 
 
 def assert_results(html, tests=1, duration=None, passed=1, skipped=0, failed=0,
-                   errors=0, xfailed=0, xpassed=0, rerun=0):
+                   errors=0, xfailed=0, xpassed=0, rerun=0,
+                   self_contained_html=False):
+
+    # Assert CSS link or inline parameter
+    if self_contained_html:
+        regex_css_inline = '<style>'
+        assert re.search(regex_css_inline, html) is not None
+    else:
+        regex_css_link = '<link href='
+        assert re.search(regex_css_link, html) is not None
+
```

Comment:
@davehunt so, my idea was creating this in assert_results, but now I think the better idea is updating only the test resources